### PR TITLE
updating 201 AML

### PR DESCRIPTION
**Pull request by ryhud**
Updating 201

### CODE REVIEW

File: quickstart/201-machine-learning-moderately-secure/workspace.tf [hunk 1]
```diff
@@ -153,5 +153,36 @@ resource "azurerm_private_endpoint" "mlw_ple" {
...
+}
+
+# Update workspace for image-build-compute
 
-}
\ No newline at end of file
+resource "null_resource" "ws_image_build_compute"{
+  provisioner "local-exec" {
+    command = <<EOF
+    az ml workspace update --resource-group ${azurerm_resource_group.default.name} --workspace-name ${azurerm_machine_learning_workspace.default.name} --image-build-compute ${azurerm_machine_learning_compute_cluster.image-builder.name}
```
Comment (denniseik) [inline quickstart/201-machine-learning-moderately-secure/workspace.tf:183]: We can update this one once the image build compute property is onboarded

File: quickstart/201-machine-learning-moderately-secure/network.tf [hunk 2]
```diff
@@ -6,11 +6,27 @@ resource "azurerm_virtual_network" "default" {
   resource_group_name = azurerm_resource_group.default.name
 }
 
-resource "azurerm_subnet" "mlsubnet" {
-  name                 = "mlsubnet"
+resource "azurerm_subnet" "training-subnet" {
```
Comment (denniseik) [inline quickstart/201-machine-learning-moderately-secure/network.tf]: We should use naming conventions per https://docs.microsoft.com/en-us/azure/cloud-adoption-framework/ready/azure-best-practices/resource-abbreviations to get it through the Terraform review.

**Review by denniseik — CHANGES_REQUESTED** [review]
This looks great! We'd need minor changes to comply with naming conventions.

File: quickstart/201-machine-learning-moderately-secure/network.tf (hunk 2 above)
Comment (denniseik) [reply quickstart/201-machine-learning-moderately-secure/network.tf]: snet-training
snet-ml
etc.

Comment (ryhud) [reply quickstart/201-machine-learning-moderately-secure/network.tf]: added proper naming conventions to resources

Comment (denniseik) [reply quickstart/201-machine-learning-moderately-secure/network.tf]: thank!

**Review by denniseik — APPROVED** [review]

**Pull request merged**